tests/cnf/ran/powermanagement/tests/cpufreq.go

```diff
@@ -0,0 +1,100 @@
+package tests
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
+	"github.com/openshift-kni/eco-goinfra/pkg/nto" //nolint:misspell
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/cluster"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/powermanagement/internal/helper"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/powermanagement/internal/tsparams"
+	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+)
+
+var _ = Describe("Per-core runtime power states tuning", Label(tsparams.LabelCPUFrequency), func() {
+	var (
+		nodeList                []*nodes.Builder
+		perfProfile             *nto.Builder
+		err                     error
```

Comment:
is this needed?

Reply:
Kirsten pointed out that it is needed, though not in the global variables. If := is used when getting the perfProfile, the perfProfile var is only scoped to within the beforeEach which causes problems when this var is used later.

Reply:
yes thats what I meant as well. It is not needed as a shared/global var.  Thanks for the change.